package.json

```diff
@@ -3,6 +3,7 @@
   "license": "AGPL-3.0",
   "scripts": {
     "postversion": "git push --tags",
+    "build:babel": "babel app/javascript --out-dir ./build",
```

Comment:
Surely this should be "build" running a webpack build? I'm not sure babel alone will handle everything given there are lots of requires.